src/delivery/utils/mod.rs

```diff
@@ -71,6 +72,22 @@ pub fn home_dir(to_append: &[&str]) -> Result<PathBuf, DeliveryError>
    }
 }
 
+// Read from STDIN
+//
+// Useful helper method to ask questions to the end-user
+//
+// Example:
+// ```
+// say("yellow", "How cool is the delivery-cli? [1-10] ");
+// let coolness = utils::read_from_terminal()?;
+// assert_eq!(coolness, 10);
+// ```
+pub fn read_from_terminal() -> DeliveryResult<String> {
```

Comment:
Is this function called anywhere?  If not, do we need to keep it?  

One note is that any time we prompt on the command line, we almost surely also want a way to prevent the prompting via command line flags.